.github/workflows/main.yml

```diff
@@ -0,0 +1,136 @@
+name: Build
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  CHART_DIR: "helm/alfresco-process-application"
+  # TODO: create dedicated?
+  CLUSTER_NAME: activiti-test
```

Comment:
@gionn is there a cluster name I can use on Alfresco repositories for testing, please?

Reply:
you can spin up a new env on the pipeline and then use that here

Reply:
ok, I think I'll do that a bit later, when the PR is ready for review :+1: 
(the activiti-test one seems to be ok to validate changes for now)

Reply:
since this cluster was used by Travis, will keep on using it with GH actions if that's alright